src/libopensc/card-isoApplet.c

```diff
@@ -67,6 +71,23 @@ static struct sc_card_driver isoApplet_drv =
 	NULL, 0, NULL
 };
 
+static struct isoapplet_supported_ec_curves {
+		struct sc_object_id oid;
+		size_t size;
+		unsigned int min_applet_version;
+} ec_curves[] = {
+	{{{1, 2, 840, 10045, 3, 1, 1, -1}},     192, 0x0000}, /* secp192r1, nistp192, prime192v1, ansiX9p192r1 */
```

Comment:
The OID is a bijective identifier for a curve - the name is not. Generally, I find it (arguably) preferable to use the OID instead of the name to reference a curve.

`sc_pkcs15_fix_ec_parameters()`in pkcs15-publey.c can be used for a mapping of names to OIDs and field length. But there is information that is duplicated in pkcs11-tool, see #398. 

If we are talking about refactoring this, maybe this can be extended to provide some form of explicit parameters for pkcs15-init drivers?